spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -124,12 +124,54 @@ interface Packet {
 
 Note that a `Packet` is never directly serialised. Rather it is an intermediary structure used in certain function calls that may need to be created or processed by modules calling the IBC handler.
 
+Question: why is this here? Isn't there a protobuf definition for Packet serialization which must be understood by all
+chains that participate in IBC?
+
 An `OpaquePacket` is a packet, but cloaked in an obscuring data type by the host state machine, such that a module cannot act upon it other than to pass it to the IBC handler. The IBC handler can cast a `Packet` to an `OpaquePacket` and vice versa.
 
 ```typescript
 type OpaquePacket = object
 ```
 
+**Update:**
+
+The first version of IBC considered `Acknowledgement` simple application dependent bytes. This stood in contrast to the
+`Packet` types, which includes a number of well-defined metadata fields in addition to an application-dependent
```

Comment:
No, these work the same way. The _contents_ of a packet may be well-defined, e.g. by ICS 20, but not by core IBC.